package.json

```diff
@@ -22,6 +22,9 @@
     "url": "https://github.com/infinum/emotion-normalize/issues"
   },
   "homepage": "https://github.com/infinum/emotion-normalize#readme",
+  "scripts": {
+    "snippet": "./bin/build.js"
+  },
   "devDependencies": {
     "babel-eslint": "^8.2.1",
     "normalize.css": "latest"
```

Comment:
Setting a "latest" version of a dependency is a bad idea in general, and since we're depending on an undocumented API (location of the css file), we should fix the dependency on the exact version. Future improvement would be to read the main field in the normalize.css package.json

Reply:
Also, you don't need emotion and react-emotion as dependencies. You don't want to force the user to install react-emotion if they don't plan to use it, and emotion should go into the peerDependencies section (and devDependencies if needed for development)

Reply:
I've fixed this. What else should be fixed with the scripts?